test/data/devicegraphs/windows-pc-raid1.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
```

Comment:
NP: For this trivial scenarios, I would use yml files (easier to read, write and maintain), see for example: https://github.com/yast/yast-storage-ng/blob/master/test/data/devicegraphs/md_raid.yml

Reply:
In fact, the test could be confusing because a Software RAID is used, but the error should not happen with Software RAIDs (Mds are not considered disk devices, so the proposal does not use them).

Why not to use the xml file from the logs for this regression test?

Reply:
This **is** the xml from the logs.

Reply:
The yml did not work. Now that I looked at it a bit closer I think the yml reader has a bug.

Reply:
Ok, thanks.